hello.java

```diff
@@ -0,0 +1,5 @@
+public class Hello {
```

Comment:
Name of public class must exactly match name of file containing it; your file is lowercase hello.